src/Utility/Text.php

```diff
@@ -564,124 +565,241 @@ public static function tail($text, $length = 100, array $options = [])
     public static function truncate($text, $length = 100, array $options = [])
     {
         $default = [
-            'ellipsis' => '...', 'exact' => true, 'html' => false
+            'ellipsis' => '...', 'exact' => true, 'html' => false, 'trimWidth' => false,
```

Comment:
Can we maybe have another method called `Text::truncateWidth()` that calls this method with this option enabled? That way we can communicate better that this is possible